src/components/adaptive-cards/ActionSet/ActionSet.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+import Component, {acPropTypes, registerComponent} from '../Component/Component';
+
+/**
+ * Adaptive Cards ActionSet component
+ * https://adaptivecards.io/explorer/ActionSet.html
+ *
+ * @param {object} props  React props passed to the component
+ * @param {object} props.data  Active cards definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Custom style to apply
+ * @returns {object} JSX of the component
+ */
+export default function ActionSet({data, className, style}) {
+  const [cssClasses] = webexComponentClasses('action-set', className);
```

Comment:
'adaptive-cards-action-set', so that we don't interphere with "standard" generic components